stage8/00-license-generation/00-run.sh

```diff
@@ -261,14 +261,63 @@ echo "<th>Location</th>" >> ${FILE}
 echo "</tr>" >> ${FILE}
 echo "</thead>" >> ${FILE}
 echo "<tbody>" >> ${FILE}
-package_table_items $((var++)) "Linux" "6.1" "GPLv2" "https://github.com/analogdevicesinc/linux"
-package_table_items $((var++)) "HDL"   " "    "GPL/LGPL/BSD" "https://github.com/analogdevicesinc/hdl"
-package_table_items $((var++)) "LibIIO" " " "LGPL-2.1/GPL-2.0" "https://github.com/analogdevicesinc/libiio"
-package_table_items $((var++)) "IIO Oscilloscope" " " "GPL-2.0" "https://github.com/analogdevicesinc/iio-oscilloscope"
-package_table_items $((var++)) "Scopy" " " "GPL-3.0" "https://github.com/analogdevicesinc/scopy"
+
+package_table_items $((var++)) "linux" "6.1" "GPLv2" "https://github.com/analogdevicesinc/linux"
+wget https://raw.githubusercontent.com/analogdevicesinc/linux/main/COPYING
+html_pre_file COPYING linux
+rm COPYING
+
+package_table_items $((var++)) "hdl"   " "    "GPL/LGPL/BSD" "https://github.com/analogdevicesinc/hdl"
+wget https://raw.githubusercontent.com/analogdevicesinc/hdl/main/LICENSE
+html_pre_file LICENSE hdl
+rm LICENSE
+
+package_table_items $((var++)) "libiio" " " "LGPL-2.1/GPL-2.0" "https://github.com/analogdevicesinc/libiio"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/libiio/COPYING.txt libiio
+
+package_table_items $((var++)) "iio-oscilloscope" " " "GPL-2.0" "https://github.com/analogdevicesinc/iio-oscilloscope"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/iio-oscilloscope/LICENSE iio-oscilloscope
+
+package_table_items $((var++)) "scopy" " " "GPL-3.0" "https://github.com/analogdevicesinc/scopy"
```

Comment:
I think that for Scopy we can set the version here (since we are adding in Kuiper just released ones, even Flatpack, even AppImage)